### PR TITLE
Implement feature described in issue #13

### PR DESCRIPTION
**Pull request by devGTP**
This PR implements the feature described in issue #13. The code was generated using devGPT, a GitHub bot based on GPT-4.

### CODE REVIEW

File: basics_character.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        elif event.type == 'KEYDOWN':
+            move = handle_keys()
+            if move:
+                clear_player(con, player_x, player_y)
+                player_x, player_y = move_player(player_x, player_y, move[0], move[1])
+                draw_player(con, player_x, player_y)
+    return player_x, player_y
+
+def main():
+    tcod.console_set_custom_font('arial10x10.png', tcod.FONT_TYPE_GREYSCALE | tcod.FONT_LAYOUT_TCOD)
```
Comment (petvas) [inline basics_character.py]: terminal display logic should not be in the character file.

File: basics_character.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    tcod.console_init_root(80, 50, title='Basics Character and Movement', fullscreen=False)
+    con = tcod.console.Console(80, 50)
+    player_x, player_y = 40, 25
+    while True:
+        tcod.console_flush()
+        clear_player(con, player_x, player_y)
+        player_x, player_y = handle_events(con, player_x, player_y)
+        draw_player(con, player_x, player_y)
+
+if __name__ == '__main__':
```
Comment (petvas) [inline basics_character.py:170]: We don't want to run this file itself, remove this.

**Review by petvas — CHANGES_REQUESTED** [review]

**Comment by devGTP** [comment]
Changes have been made based on the review comments. Please review the updated code.

File: basics_character.py [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+                clear_player(con, player_x, player_y)
+                player_x, player_y = move_player(player_x, player_y, move[0], move[1])
+                draw_player(con, player_x, player_y)
+    return player_x, player_y
+
+def main():
+    con = tcod.console.Console(80, 50)
+    player_x, player_y = 40, 25
+    while True:
+        tcod.console_flush()
```
Comment (petvas) [inline basics_character.py:43]: move this logic to the main.py file

**Review by petvas — CHANGES_REQUESTED** [review]

**Comment by devGTP** [comment]
Changes have been made based on the review comments. Please review the updated code.

**Pull request closed without merging**